CMakeLists.txt

```diff
@@ -1,13 +1,36 @@
-# This file is used for testing only
+project(editorconfig-core-qt)
+cmake_minimum_required(VERSION 3.20)
```

Comment:
Should we re-order? And `cmake_minimun_required()` be on top? This way I can add metadata to the project.

Reply:
The few other cmake files usually has cmake_minimun_required() on top, so I think this would be a good idea.